admin/authentication/local-strategy.js

```diff
@@ -6,6 +6,7 @@ const logger = require('../services/log.service').getLogger()
 
 const userDataService = require('../services/data-access/user.data.service')
 const adminLogonEventDataService = require('../services/data-access/admin-logon-event.data.service')
+const { isNumber, isNotNil, isInteger } = require('ramda-adjunct')
```

Comment:
we should use these more.  Most of the validation across the admin app is just checking for undefined parameters